non-fungible-token/src/base.rs

```diff
@@ -0,0 +1,35 @@
+use gstd::{prelude::*, ActorId};
+use primitive_types::U256;
+
+pub trait NonFungibleTokenBase {
+    /// called during the NFT contract deployment
```

Comment:
Doc sentences start from the upper case letter and end with the dot.
```suggestion
    /// Called during the NFT contract deployment.
```